tools/bundle-size-tools/prettier.config.cjs

```diff
@@ -1,9 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-module.exports = {
```

Comment:
I don't think you meant to revert the whole file here 🙂

Reply:
Never mind. Per our conversation on Teams, this package no longer exists (just a lingering directory with an npmignore file). Looks good!